src/Microsoft.ML.Data/Transforms/OneToOneTransformerBase.cs

```diff
@@ -15,7 +15,7 @@ public abstract class OneToOneTransformerBase : RowToRowTransformerBase
     {
         protected readonly (string input, string output)[] ColumnPairs;
 
-        protected OneToOneTransformerBase(IHost host, (string input, string output)[] columns) : base(host)
+        protected OneToOneTransformerBase(IHost host, params (string input, string output)[] columns) : base(host)
```

Comment:
>params [](start = 54, length = 6)

Is this related? It might be a good idea, just not sure?

Reply:
I think I replaced one of Create methods with constructor which accept only one input and output column, and I didn't like idea what I need to cast that pair into array of pairs. 
This is why I changed it. Not sure that constructor is still in part of codereview, but I don't think it's in general bad thing to have.

---
In reply to: [245457186](https://github.com/dotnet/machinelearning/pull/2031#discussion_r245457186) [](ancestors = 245457186)

Reply:
All right, thanks Ivan.

---
In reply to: [245457798](https://github.com/dotnet/machinelearning/pull/2031#discussion_r245457798) [](ancestors = 245457798,245457186)